data/forms/child-referral.jsx

```diff
@@ -0,0 +1,565 @@
+      components: [
+        {
+          component: 'Radios',
+          name: 'professionals_details',
+          label:
+            'Do you have any details of any other professionals or agencies working with the family: ?',
+          options: ['Yes', 'No'],
+        },
+        <h3 key="text" className="govuk-body">
+          Details of any other professionals or agencies working with the
+          family:
+        </h3>,
+        <p key="text" className="govuk-body">
+          (eg GP, health visitor, other professionals working with the
+          household)
+        </p>,
+        {
+          component: 'TextInput',
+          name: 'professional_name',
+          width: '30',
+          label: 'Name of professional (1)',
+          rules: { required: true },
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_relationship',
+          width: '30',
+          label:
+            'Relationship of the professional to the child and family - professional (1)',
+        },
+
+        {
+          component: 'TextInput',
+          name: 'professional_telephone_1',
+          width: '30',
+          label: 'Telephone number - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_email_1',
+          width: '30',
+          label: 'Email address - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_address_1',
+          width: '30',
+          label: 'Address line 1 - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_address_2',
+          width: '30',
+          label: 'Address line 2 - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_address_3',
+          width: '30',
+          label: 'Address line 3 - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_postcode_1',
+          width: '30',
+          label: 'Postcode - professional (1)',
+        },
+        {
+          component: 'Radios',
+          name: 'professional_consent',
+          label:
+            'Professional (1): Has the person with parental responsibility consented for Hackney to make contact with this person or agency?',
+          options: ['Yes', 'No'],
+        },
+        {
+          component: 'Radios',
+          name: 'professional_other',
+          label:
+            'Do you have any more information about additional professionals working with the family',
+          options: ['Yes', 'No'],
+          rules: { required: true },
+        },
+      ],
+    },
+    {
+      id: 'requestForSupport',
+      title: 'Request For Support',
+      components: [
+        <h3 key="text" className="govuk-body">
+          Hackney Child Wellbeing Framework
+        </h3>,
+        <p key="text" className="govuk-body">
```

Comment:
We should avoid having list elements with the same `key`. (reference https://reactjs.org/docs/lists-and-keys.html#keys)

Reply:
Does this mean every piece of stand alone text should have a separate key or we leave the key out?

Reply:
Added unique key for each standalone text tag